### PR TITLE
Fixes: #4890 to support new ssh key types

### PR DESCRIPTION
**Pull request by punkrokk**
Fixes: #4890 to support new Paramiko keys. 

Notes: 
Paramiko Issue: https://github.com/paramiko/paramiko/issues/1565 (released in v2.7.1)
Changelog in v2.7.1: http://www.paramiko.org/changelog.html

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -53,6 +53,7 @@ Changed
   connection related errors, our code would first wait for this timeout to be reached (30 seconds)
   before returning error to the end user. #4834
 * Upgrade ``pymongo`` to the latest stable version (``3.10.0.``). #4835 (improvement)
+* Updated Parmiko to v2.7.1 to support new key formats #4891 (improvement)
```
Comment (armab) [inline CHANGELOG.rst:56]: Would be nicer to note _what_ kind of keys are now supported with this update.

Comment (armab) [reply CHANGELOG.rst:56]: #4891 in Changelog references unrelated PR. I guess #4901 should be used instead.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -52,6 +52,7 @@ Changed
...
```
Comment (nmaludy) [inline CHANGELOG.rst:55]:
```suggestion
* Updated Parmiko to v2.7.1 to support new key formats #4901 (improvement)
```
Created a suggestion out of @armab comment so all you should have to do is hit the "Accept Suggestion" button.

**Comment by nmaludy** [comment]
@armab FYI i created a suggestion and added it to this PR that fixes the referenced version in the CHANGELOG. Should be good to go now.

**Comment by punkrokk** [comment]
I think he wanted the names of the keys. Haven’t gotten to it yet.

**Comment by punkrokk** [comment]
@armab before we merge, I don't understand how the .travis file ended up here

**Comment by punkrokk** [comment]
@blag I think you accidentally commit the travis file here.

**Comment by armab** [comment]
I'm just happy @blag didn't accidentally commit more `pip` changes to this PR :smile:

**Comment by blag** [comment]
@armab I had to resist the urge! 😜

**Pull request merged**